test/functional/dataElements/xdmObject/xdmObjectFieldValidation.spec.js

```diff
@@ -10,72 +10,58 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import initializeExtensionView from "./xdmObject/helpers/initializeExtensionView";
-import xdmTree from "./xdmObject/helpers/xdmTree";
-import arrayEdit from "./xdmObject/helpers/arrayEdit";
-import stringEdit from "./xdmObject/helpers/stringEdit";
-import spectrum from "../helpers/spectrum2";
-import createExtensionViewController from "../helpers/createExtensionViewController";
-
-const schemaTitle = "XDM Object Data Element Tests";
-const schemaSelectField = spectrum.combobox("schemaField");
-const selectSchemaFromSchemasMeta = async () => {
-  await schemaSelectField.selectOption(schemaTitle);
-};
-
-const extensionViewController = createExtensionViewController(
-  "dataElements/xdmObject.html"
-);
+import initializeExtensionView from "./helpers/initializeExtensionView";
+import xdmTree from "./helpers/xdmTree";
+import arrayEdit from "./helpers/arrayEdit";
+import stringEdit from "./helpers/stringEdit";
+import editor from "./helpers/editor";
 
 // disablePageReloads is not a publicized feature, but it sure helps speed up tests.
 // https://github.com/DevExpress/testcafe/issues/1770
-fixture("XDM Object Population Indicator")
+fixture("XDM Object Validation")
   .disablePageReloads.page("http://localhost:3000/viewSandbox.html")
   .meta("requiresAdobeIOIntegration", true);
 
 test("arrays with no values are invalid", async () => {
-  await initializeExtensionView();
-  await selectSchemaFromSchemasMeta();
```

Comment:
If/When we add more schemas to our sandbox, we might need to add something like `selectSchemaFromSchemasMeta` back in, but right now we only have one schema in the sandbox so it gets automatically selected.